partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/WhitelistedTests.txt

```diff
@@ -3070,3 +3070,9 @@ run/virtpatmat_switch.scala
 run/t5629b.scala
 run/t6318_primitives.scala
 run/t8764.scala
+
+# Big numerics added
```

Comment:
Don't put the blank line nor the comment. We don't put comments in `WhiteListedTests.txt`.